packages/cactus-plugin-satp-hermes/src/main/go/generated/gateway-client/docs/TransactRequest.md

```diff
@@ -7,18 +7,20 @@ Name | Type | Description | Notes
 **ContextID** | **string** |  | 
```

Comment:
Please add considerations/documentation on the BLO and BLO, transact service on the documentation. Thanks!